user-api/overlays/moc-prod/deploymentconfig.yaml

```diff
@@ -126,6 +126,11 @@ spec:
                 secretKeyRef:
                   name: ceph
                   key: secret-key
+            - name: THOTH_SEARCH_UI_URL
+              valueFrom:
+                secretKeyRef:
+                  name: thoth-search-ui-url
+                  key: "https://thoth-station.ninja/search/"
```

Comment:
Is this the key we want to use for the secret?

Reply:
Right thank you (copy-paste error), I will replace it with `configMapKeyRef` 👍🏻